doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2472 @@
+
+
+The ``savetxt()`` and ``loadtxt()`` functions accept additional optional parameters such as header, footer, and delimiter. While text files can be easier for sharing, .npy and .npz files are faster to retrieve. If you need more sophisticated handling of your text file (for example, if you need to work with lines that contain missing values), you will want to use the  `genfromtxt function <https://numpy.org/devdocs/reference/generated/numpy.genfromtxt.html>`_.
+
+With ``savetxt``, you can specify headers, footers, comments, and more. `Read more about savetxt here <https://numpy.org/devdocs/reference/generated/numpy.savetxt.html>`_.
+
+You can read more about `save <https://numpy.org/devdocs/reference/generated/numpy.save.html>`_ here, `savez <https://numpy.org/devdocs/reference/generated/numpy.savez.html>`_ here, and `load <https://numpy.org/devdocs/reference/generated/numpy.load.html>`_ here. 
+You can read more about `savetxt <https://numpy.org/devdocs/reference/generated/numpy.savetxt.html>`_ here, and `loadtxt <https://numpy.org/devdocs/reference/generated/numpy.loadtxt.html>`_ here.
+
+Learn more about `input and output routines here <https://numpy.org/devdocs/reference/routines.io.html>`_.
+
+
+Importing and exporting a CSV
+-----------------------------
+
+It's simple to read in a CSV that contains existing information. The best and easiest way to do this is to use `Pandas <https://pandas.pydata.org/getpandas.html>`_.
+
+::
+
+  import pandas as pd
+
+  # If all of your columns are the same type:
+  x = pd.read_csv('music.csv').values
+
+  # You can also simply select the columns you need:
+  x = pd.read_csv('music.csv', columns=['float_colname_1', ...]).values
+
+.. image:: images/np_pandas.png
+
+It's simple to use Pandas in order to export your array as well. If you are new to NumPy, you may want to  create a Pandas dataframe from the values in your array and then write the data frame to a CSV file with Pandas.
+
+If you created this array "a"
+
+::
+
+  [[-2.58289208,  0.43014843, -1.24082018,  1.59572603],
+   [ 0.99027828,  1.17150989,  0.94125714, -0.14692469],
+   [ 0.76989341,  0.81299683, -0.95068423,  0.11769564],
+   [ 0.20484034,  0.34784527,  1.96979195,  0.51992837]]
+
+You could create a Pandas dataframe
+
+::
+
+  df = pd.DataFrame(a)
+  print(df)
+
+**Output:**
+
+::
+
+              0         1         2         3
+    0 -2.582892  0.430148 -1.240820  1.595726
+    1  0.990278  1.171510  0.941257 -0.146925
+    2  0.769893  0.812997 -0.950684  0.117696
+    3  0.204840  0.347845  1.969792  0.519928
+
+You can easily save your dataframe with:
+
+::
+
+  df.to_csv('pd.csv')
+
+And read your CSV with:
+
+::
+
+  pd.read_csv('pd.csv')
+
+.. image:: images/np_readcsv.png
+
+You can also save your array with the NumPy ``savetxt`` method.
+
+::
+
+  np.savetxt('np.csv', a, fmt='%.2f', delimiter=',', header='1,  2,  3,  4')
+
+Read your saved CSV any time with a command such as:
+
+**Input:**
+
+::
+
+  cat np.csv
```

Comment:
Perhaps worth noting this only works on linux

Reply:
It's also not clear that this command is meant to be executed on the command line rather than in a Python terminal. Better to spell that out. And to address the first comment, how about adding "... or open the file with a text editor"?

Reply:
Got it!